finance/ledger/Portfile

```diff
@@ -18,18 +21,19 @@ maintainers         nomaintainer
 
 platforms           darwin
 
-checksums           rmd160  ff4502d36d80b4b5e4e3b824e3148a4d771273cd \
-                    sha256  47b474c3c29029191c7f05b7d228becaa8e2ba8221d6968fa0b5fd9241ff10fe \
-                    size    800631
+checksums           rmd160  6286315fcad52f51c59f9b41261713aa141e9fae \
+                    sha256  92bf09bc385b171987f456fe3ee9fa998ed5e40b97b3acdd562b663aa364384a \
+                    size    790959
 
-depends_lib-append  port:boost \
-                    port:python27 \
+depends_lib         port:boost \
```

Comment:
Change `depends_lib` back to `depends_lib-append` so you don't overwrite the dependencies from the portgroups.

Reply:
Thanks. Done!